### DISCUSSION TITLE
SO auth callback

### DISCUSSION THREAD

**Discussion by younes200** [Budibase/budibase#8834] [Help] [answered]
Hi,
Callback URL in auth section shows 'http://localhost:10000/api/global/auth/google/callback' I tried to set the `PLATFORM_URL` env variable but doesn't seems to have effect. Do you know how I can change this ?
Thank you in advance.

**Comment by andz-bb** [comment]
hey @younes200, what is the value for `Platform URL` showing as in the Organisation section of your settings?

**Comment by younes200** [comment, marked as answer]
Hello @andz-bb 
it was still on `http://localhost:10000` change it works fine now !
Thanks for your help.
/Y
